cpp/include/cugraph/algorithms.hpp

```diff
@@ -2349,6 +2349,25 @@ rmm::device_uvector<vertex_t> maximal_independent_set(
   graph_view_t<vertex_t, edge_t, false, multi_gpu> const& graph_view,
   raft::random::RngState& rng_state);
 
+/*
+ * @brief Find a Greedy Coloring using Maximal Independent Set
+ *
+ * @tparam vertex_t Type of vertex identifiers. Needs to be an integral type.
+ * @tparam edge_t Type of edge identifiers. Needs to be an integral type.
+ * @tparam multi_gpu Flag indicating whether template instantiation should target single-GPU (false)
+ * @param handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator, and
+ * handles to various CUDA libraries) to run graph algorithms.
+ * @param graph_view Graph view object.
+ * @param rng_state The RngState instance holding pseudo-random number generator state.
+ * @return A device vector containing color for each vertex
+ */
+
```

Comment:
No need for an empty line here. This is inconsistent with the other algorithm declarations.

Reply:
And based on https://en.wikipedia.org/wiki/Graph_coloring, `vertex_coloring` could be more descriptive name than just `coloring`.

Reply:
Is there any paper describing the algorithm behind your coloring implementation? If yes, better list it here.